src/styles/sidebar/components/annotation-publish-control.scss

```diff
@@ -59,6 +53,20 @@
     &:hover {
       background-color: var.$grey-6;
     }
+
+    // This selects for the `.menu` child div. Make sure it uses the full
+    // available height in the `publish-button` parent
+    & > div {
```

Comment:
By making this selector for immediate-child `div`s, I'm hoping it's less offensive than selecting for `div.menu` specifically. The notion being that we want to "make any child div the full height of the parent".